app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,10 @@
+class OrdersController < ApplicationController
+  def create
+    @client = current_user
+    @line_items = @client.order.last.line_items
```

Comment:
I'm a little confused what's going on here, which may be on me. I'm not really sure what @line_items is trying to do - also I'm confused in general by an instance variable that's pluralized like that? If it's an instance isn't it almost definitionally a single.. instance of a line item?